docs/core/PORTING.md

```diff
@@ -570,58 +556,58 @@ As shown in the snippet above, the uVisor needs the following regions:
-        It holds the monolithic uVisor core binary. Its location determines the API table of uVisor, since <code>uvisor_api</code> points to the first entry in this table. This region must be located at the same offset that you specified as <code>FLASH_OFFSET</code> in your configuration.
+        It holds the monolithic uVisor core binary. Its location determines the API table of uVisor because <code>uvisor_api</code> points to the first entry in this table. This region must be located at the same offset as <code>FLASH_OFFSET</code> in your configuration.
       </td>
     </tr>
     <tr>
       <td>
         <code>.uvisor.bss</code>
       </td>
       <td>
-        It contains both uVisor's own memories (coming from the uVisor library, in <code>.keep.uvisor.bss.main</code>) and the private boxes' protected memories (in <code>.keep.uvisor.bss.boxes</code>). This region must be located at the same offset that you specified in <code>SRAM_OFFSET</code>. If uVisor shares the SRAM with the OS/app, this region must be positioned right before the page heap section, and immediately after the VTOR relocation section. Otherwise, it spans the whole memory devoted to it (e.g. a TCM).
+        It contains uVisor's memories (coming from the uVisor library, in <code>.keep.uvisor.bss.main</code>) and the private boxes' protected memories (in <code>.keep.uvisor.bss.boxes</code>). This region must be located at the same offset that you specified in <code>SRAM_OFFSET</code>. If uVisor shares the SRAM with the OS/app, this region must be positioned right before the page heap section and immediately after the VTOR relocation section. Otherwise, it spans the whole memory devoted to it (for example, a TCM).
       </td>
     </tr>
     <tr>
       <td>
         <code>.page_heap</code>
       </td>
       <td>
-        It is needed to host pages allocated by the uVisor secure allocator. If the uVisor shares the SRAM with the OS/app, it must be placed right after the uVisor BSS section, otherwise it must be the first SRAM region after the VTOR relocation section.
+        It hosts pages the uVisor secure allocator allocates. If the uVisor shares the SRAM with the OS/app, it must be immediately after the uVisor BSS section. Otherwise, it must be the first SRAM region after the VTOR relocation section.
       </td>
     </tr>
     <tr>
       <td>
         <code>.uvisor.secure</code>
       </td>
       <td>
-        It contains constant data in flash that describes the configuration of the secure boxes. It comprises the configuration tables (<code>.keep.uvisor.cfgtbl</code>) and the pointers to them (<code>.keep.uvisor.cfgtbl_ptr[_first]</code>), which are used by uVisor for box enumeration.
+        It contains constant data in flash that describes the configuration of the secure boxes. It comprises the configuration tables (<code>.keep.uvisor.cfgtbl</code>) and the pointers to them (<code>.keep.uvisor.cfgtbl_ptr[_first]</code>), which uVisor uses for box enumeration.
```

Comment:
Does `uvisor.secure` make up the configuration tables and pointers to them, or do the configuration tables and pointers make up `uvisor.secure`?